docs/documentation/stories/rc.0-update.md

```diff
@@ -311,20 +311,23 @@ We've updated a lot of packages over the last months in order to keep projects u
 
 Additions or removals are found in bold below.
 
+`"angular-cli": {},` was **removed**
```

Comment:
What beta was that though? This guide only covers from beta.24 up and I didn't see that entry there.